app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlAdapter.kt

```diff
@@ -38,8 +38,26 @@ import mozilla.components.feature.tab.collections.Tab as ComponentTab
 
 sealed class AdapterItem(@LayoutRes val viewType: Int) {
     data class TipItem(val tip: Tip) : AdapterItem(
-        ButtonTipViewHolder.LAYOUT_ID)
-    data class TopSiteList(val topSites: List<TopSite>) : AdapterItem(TopSiteViewHolder.LAYOUT_ID)
+        ButtonTipViewHolder.LAYOUT_ID
+    )
+
+    data class TopSiteList(val topSites: List<TopSite>) : AdapterItem(TopSiteViewHolder.LAYOUT_ID) {
+        override fun getChangePayload(newItem: AdapterItem): Any? {
```

Comment:
`getChangePayload` should have a comment mentioning the item type is always the same as the class type